### PR TITLE
Deploy helloworld on prod cluster

### PR DESCRIPTION
**Pull request by ca-schue**
Currently reachable under https://dev.universphere.cloud/helloworld/

### CODE REVIEW

File: kubernetes/production-cluster/helloworld/helloworld-ingress.yaml [hunk 1]
```diff
@@ -1,22 +1,24 @@
...
-    http:
-      paths:
-      - path: /
-        pathType: Prefix
-        backend:
-          service:
-            name: helloworld-service
-            port:
-              number: 80
+    - host: dev.universphere.cloud
```
Comment (shei99) [inline kubernetes/production-cluster/helloworld/helloworld-ingress.yaml:10]:
```suggestion
    - host: universphere.cloud
```

**Review by shei99 — COMMENTED** [review]
Please deploy it to the production cluster. If it is available there, ill approve this PR.

File: kubernetes/production-cluster/helloworld/helloworld-ingress.yaml [hunk 2]
```diff
@@ -1,22 +1,24 @@
...
+          - pathType: Prefix
+            path: /helloworld
+            backend:
+              service:
+                name: helloworld
+                port:
+                  number: 80
+  tls:
+    - hosts:
+        - dev.universphere.cloud
```
Comment (ca-schue) [inline kubernetes/production-cluster/helloworld/helloworld-ingress.yaml:22]:
```suggestion
        - universphere.cloud
```

**Comment by ca-schue** [comment]
Reachable under "https://universphere.cloud/helloworld/", waiting for commit of @mansourm97

**Comment by D4ykoo** [comment]
lgtm!

**Pull request merged**